core/src/main/scala/org.broadinstitute.dsde.workbench.leonardo/package.scala

```diff
@@ -0,0 +1,6 @@
+package org.broadinstitute.dsde.workbench
+
+package object leonardo {
+  type LabelMap = Map[String, String]
```

Comment:
Maybe we should get rid of `LabelMap`

Reply:
I'd be happy to do that :)

Reply:
actually, I think removing it makes some type a bit too long. Like ` Either[ParseLabelsException, LabelMap] `